package.json

```diff
@@ -14,26 +14,21 @@
         "@babel/preset-env": "^7.25.0",
         "@babel/preset-typescript": "^7.24.7",
         "@types/k6": "^0.52.0",
-        "@types/webpack": "^5.28.5",
         "@typescript-eslint/eslint-plugin": "^7.17.0",
         "@typescript-eslint/parser": "^7.17.0",
         "babel-loader": "^9.1.3",
         "chai": "5.1.1",
-        "clean-webpack-plugin": "^4.0.0",
-        "copy-webpack-plugin": "^12.0.2",
+        "esbuild": "^0.23.1",
         "eslint": "^8.56.0",
         "eslint-plugin-import": "^2.29.1",
-        "terser-webpack-plugin": "^5.3.10",
-        "typescript": "^5.5.4",
-        "webpack": "^5.93.0",
-        "webpack-cli": "^5.1.4",
-        "webpack-glob-entries": "^1.0.1"
+        "typescript": "^5.5.4"
     },
     "engines": {},
     "scripts": {
-        "webpack": "./node_modules/.bin/webpack",
```

Comment:
If I'm not wrong, looks like the Release workflow is still using this command (see https://github.com/grafana/k6-jslib-aws/blob/main/.github/workflows/release.yml#L25). So, I guess that should be adjusted in this PR as well, right? 🤔

Reply:
Good catch 👀 On it 🙇🏻